tests/models/multimodal/generation/test_common.py

```diff
@@ -308,6 +308,37 @@
         stop_str=["<｜end▁of▁sentence｜>", "<｜begin▁of▁sentence｜>"],  # noqa: E501
         image_size_factors=[(), (1.0, ), (1.0, 1.0, 1.0), (0.1, 0.5, 1.0)],
     ),
+    "eagle2_5_vl": VLMTestInfo(
+        models=[
+            "nvidia/Eagle2.5-8B",
+        ],
+        test_type=(VLMTestType.IMAGE,VLMTestType.MULTI_IMAGE),
```

Comment:
```suggestion
        test_type=(VLMTestType.IMAGE, VLMTestType.MULTI_IMAGE, VLMTestType.VIDEO),
```
We can consolidate video test case.